learn/glossary.md

```diff
@@ -73,16 +129,62 @@ A static URI (or IP) that you have assigned to your node, which applications can
 
 ## Session
 
-The relationship between an Application and the Validator(s) that service it at any point in time.
+A data structure which uses data from the finality storage layer of the network to pseudo-randomly group one application with a set of nodes to provide service for a limited time. A session lasts for 4 blocks, for a total of one hour.
 
-Every Application will only have one single **Session** per Relay Chain it is staked for. Meaning following an Application stake event, there will _always_ be a corresponding **Session** for the Application for each Relay Chain until it unstakes.
+Every application will only have one single session per relay chain it is staked for. Meaning, following an application stake event, there will always be a corresponding session for the application for each relay chain until it unstakes.
 
 ## Session Block
 
 Some number of blocks, marking session beginning/ends.
 
+## Stickiness
+
+A feature that lets you connect to the same node for a set period of time. With stickiness all the relays are sent to the same node, ensuring more consistent relays.
+
+## Trophy
+
+Achievement earned for proving knowledge of and participation in the Pocket Network ecosystem. These can be collected and used to earn a vote in the Pocket DAO. There are [four distinct paths to a vote](https://docs.pokt.network/home/community/trophies), each with different trophies: App Developers, Node Runners, Community Shepherds, and Governors/Contributors.
+
+## v0
+
+Current active iteration of the Pocket protocol. Often used in contrast to the upcoming v1 update.
+
+## v1
+
+Pocket v1 (or just "v1") is a [major planned overhaul of the Pocket Network protocol](future/README.md), including a departure from Tendermint. v1 will allow Pocket to build its own network from the ground up, optimized for its specific use case, rather than being restricted by any limitations that come with building on Tendermint. v1 will bring major changes to four different "modules": Utility, Consensus, Peer-to-Peer, and Persistence.
+
 ## Validator
 
-**Validators** are responsible for committing new blocks in the blockchain. These **validators** participate in the consensus protocol by broadcasting votes which contain cryptographic signatures signed by each **validator’s** private key.
+Special nodes that are responsible for committing new blocks in the blockchain. Validators participate in the consensus protocol by broadcasting votes which contain cryptographic signatures signed by each validator's private key.
+
+Validators stake POKT into the protocol to participate in Servicing and Consensus, just like Servicers, but because they are able to be block proposers, they can earn a slightly higher reward amount, as determined by the [ProposerAllocation](protocol-parameters.md#proposerallocation) parameter.
```

Comment:
Each validator has a chance to propose the next block proportional to the amount that they have staked. If that block is accepted, they are rewarded POKT for doing so. The amount of POKT they earn is determined by the ProposerAllocation parameter